infrastructure/server-setup/playbook-1.yml

```diff
@@ -133,6 +109,27 @@
       script: ../cryptfs/mount.sh -p {{encrypt_passphrase}}
       when: st.stat.exists and encrypt_data
 
+    - name: Copy reboot.service systemd file
+      ansible.builtin.copy:
+        dest: /etc/systemd/system/reboot.service
+        group: 1000
+        owner: 1000
+        mode: g+rwx
+        content: |
+          [Unit]
+          Description=Mount encrypted dir
+
+          [Service]
+          ExecStart=bash /opt/opencrvs/infrastructure/cryptfs/reboot.sh -pF /data/secrets/opencrvs.secrets >> /var/log/cryptfs-reboot.log 2>&1
```

Comment:
Can the `/data/secrets/opencrvs.secrets` file be read before the `/data` drive is decrypted?

Reply:
No it cant.  But I'm not sure that is a problem tbh.  Is there anything you think might be a specific challenge there?

Reply:
I just thought if there is a bit of a chicken and an egg situation in the reboot.sh as it expects to read that file to decrypt to disk, but it can't as the disk is not yet decrypted 

https://github.com/opencrvs/opencrvs-core/pull/3749/files#diff-a0dd732735d5b2fb220b6ee026c2e421c6f56286f68fc050f3260a5384bd59e8R39

Reply:
Ah yes, of course.

Reply:
I'm now loading the secrets commands from this playbook in Farajaland so that the approach can be customised: https://github.com/opencrvs/opencrvs-farajaland/pull/329.

Renamed my reboot script to be "decrypt.sh" and encrypt_passphrase to be: disk_encryption_key. It seemed better

I moved emergency-backup-metadata.sh and emergency-restore-metadata.sh script into Farajaland too so that they could potentially load the secrets from an HSM.  The deploy script in core will copy them over.